app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -172,6 +173,15 @@ protected function create(array $data)
 
         // Send confirmation mail.
         Mail::to($user)->queue(new \App\Mail\Confirmation($user->name));
+        // Send notification about new user to the admins and to the secretaries.
+        $admins = User::role(Role::NETWORK_ADMIN)->get();
+        $secretaries = User::role(Role::SECRETARY)->get();
+        foreach ($admins as $admin) {
+            Mail::to($admin)->send(new NewRegistration($admin->name, $user->name, $user->isCollegist()));
+        }
+        foreach ($secretaries as $secretary) {
+            Mail::to($secretary)->send(new NewRegistration($secretary->name, $user->name, $user->isCollegist()));
```

Comment:
Here passing only $user is enough and then you can handle this in the constructor.